### ISSUE TITLE
Make quicksys available in common quicklisp distributions

### ISSUE THREAD

**Issue by ryukinix** [commonlispbr/quicklisp-meta#7]
- [x] [ultralisp.org][ultra]
- [ ] official quicklisp ([pending approval here][quick])

[quick]: https://github.com/quicklisp/quicklisp-projects/issues/1654
[ultra]: https://ultralisp.org/

**Comment by luksamuk** [comment]
I suppose it might be necessary to change the project's name soon, so this is still pending.

**Comment by ryukinix** [comment]
Yes! I'll remove our project from ultradist until this can be solved.

**Comment by ryukinix** [comment]
Actually is not possible to remove anything from ultradist. What we should do about that?
